src/main/java/Task.java

```diff
@@ -25,7 +25,8 @@ public Task(String description, String type) {
      * @return Status of the task.
```

Comment:
Would be nice to have one line/brief description! I know that you describe in the return what its returning, but would be useful to maybe include which represents what?